composer.json

```diff
@@ -25,7 +25,7 @@
         "symfony/monolog-bundle"               : "^3.0",
         "symfony/polyfill-apcu"                : "^1.0",
         "symfony/swiftmailer-bundle"           : "^2.3",
-        "symfony/symfony"                      : "^3.2",
+        "symfony/symfony"                      : "dev-master",
```

Comment:
depending on dev-master directly is a total no-go.

And depending of ``^3.3@dev`` would make this PR unmergeable, as it would mean that the demo would be in a state where it cannot be released (depending on beta releases is fine, but not on the dev version)